packages/back-end/src/routers/environment/environment.controller.ts

```diff
@@ -18,6 +26,50 @@ type CreateEnvironmentResponse = {
   environment: Environment;
 };
 
+export const putEnvironmentOrder = async (
+  req: AuthRequest<{
+    environments: string[];
+  }>,
+  res: Response
+) => {
+  const context = getContextFromReq(req);
+  const { org } = context;
+  const envIds = req.body.environments;
+
+  const existingEnvs = org.settings?.environments;
+
+  if (!existingEnvs) {
+    return res.status(400).json({
+      status: 400,
+      message: "Unable to find organization's environments",
+    });
+  }
+
+  const updatedEnvs: Environment[] = [];
```

Comment:
TODO: Should I check to see if the user has permission to update envs globally or in atleast 1 project first?